xarray/tests/test_dask.py

```diff
@@ -699,3 +733,38 @@ def build_dask_array(name):
     return dask.array.Array(
         dask={(name, 0): (kernel, name)}, name=name,
         chunks=((1,),), dtype=np.int64)
+
+
+@pytest.mark.parametrize('persist', [
+    lambda x: x.persist(),
+    pytest.mark.skipif(LooseVersion(dask.__version__) < '0.16',
+                       lambda x: dask.persist(x)[0],
+                       reason='Need Dask 0.16+')
```

Comment:
This is pretty confusing at first glance, unless you already deeply understand how pytest marks work.

I don't really have a suggestion for how to make this better, but maybe a comment is in order?

Reply:
Added a comment on the first such use of this parametrization